### ISSUE TITLE
Add "AAT" to list of sources in Genre/Form term

### ISSUE THREAD

**Issue by bseeger** [jhu-idc/idc-isle-dc#85]
The Genre/Form taxonomy Authority Sources list is missing the option of The Art & Architecture Thesaurus. 

Also, remove "Other" option.

The sources available should be: 

```
  lgcft|Library of Congress Genre/Form Terms
  fast|Faceted Application of Subject Terminology
  aat|Art & Architecture Thesaurus
```

**Comment by bseeger** [comment]
addressed by https://github.com/jhu-idc/idc-isle-dc/pull/113

**Comment by htpvu** [comment]
closed by #113
